website/docs/docs/build/dimensions.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Dimensions
+id: dimensions
+description: "Dimensions determine the level of aggregation for a metric, and are non-aggregatable expressions."
+sidebar_label: "Dimensions"
+tags: [Metrics, Semantic Layer]
+---
+
+Dimensions are non-aggregatable expressions that define the level of aggregation for a metric used to define how data is sliced or grouped in a metric, Since dimensions can't be aggregated, they're considered to be a property of the primary or unique entity of the table.
```

Comment:
```suggestion
Dimensions are non-aggregatable expressions that define the level of aggregation for a metric used to define how data is sliced or grouped in a metric. Since dimensions can't be aggregated, they're considered to be a property of the primary or unique entity of the table.
```